Portfolio/index.html

```diff
@@ -71,132 +99,101 @@
 
 
 
-/* body */
-body {
-
-    background-color: #003E66;
-    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
-    color: #00D8FF;
-    margin: 0;
-    padding: 0;
 
-}
-.John{
+.abtme{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #004572;
+  
+    position: relative;
     height: 500px;
-    font-family: Revalia;
 }
- .John h1, h2{

```

Comment:
The '.abtme h2' rule should be outside of the '.abtme h1' rule to separate them properly.